includes/discovery/cisco-vrf-lite.inc.php

```diff
@@ -21,7 +21,7 @@
     $ids = [];
 
     // For the moment only will be cisco and the version 3
-    if ($device['os_group'] == 'cisco' && $device['snmpver'] == 'v3') {
+    if (isset($device['os_group']) && $device['os_group'] == 'cisco' && $device['snmpver'] == 'v3') {
         $mib = 'SNMP-COMMUNITY-MIB';
         $mib = 'CISCO-CONTEXT-MAPPING-MIB';
```

Comment:
These two lines looks like a bug :)

Reply:
haha, yeah, fix is to delete line 25, but oh well.

Reply:
cisco-vrf-lite looks like a bug :) I haven't seen any cisco device that runs this module and not standard VRF module so far ... even my oldies. I suppose this is mostly legacy code now.